### PR TITLE
Ollama doc review

### PR DESCRIPTION
**Pull request by billy-the-fish**

### CODE REVIEW

File: docs/openai.md [hunk 1]
```diff
@@ -0,0 +1,592 @@
...
+    , "date"
+    , detail
+    ) as commit_desc
+    from commit_history
+    -- just look at commits from August 2023
+    where date_trunc('month', "date") = '2023-08-01 00:00:00+00'::timestamptz
+    order by "date"
+) x
+;
+```
```
Comment (jonatas) [inline docs/openai.md:592]: I was thinking about we also put a link to the raw examples in the examples folder in the end as an encouragement to let people pick the scripts and run :)


> If you're interested in run this or  other full examples, check the [examples](../examples) folder for more.

**Review by jonatas — APPROVED** [review]
Looks really great @billy-the-fish !

I love the new organization! Looking forward to add mkdocs to deploy it somewhere too :)

File: README.md [hunk 2]
```diff
@@ -17,35 +17,40 @@ pgai brings embedding and generation AI models closer to the database. With pgai
...
+    * [OpenAI](./docs/openai.md) - configure pgai with OpenAI, then use the model to tokenize, embed, chat complete and moderate. This page also includes advanced examples.
 * **Extension contributor**: Contribute to pgai and improve the project.
-  * [Develop and test changes to the pgai extension](./DEVELOPMENT.md)
+  * [Develop and test changes to the pgai extension](./DEVELOPMENT.md).
   * See the [Issues tab](https://github.com/timescale/pgai/issues) for a list of feature ideas to contribute.
 
 **Learn more about pgai:** To learn more about the pgai extension and why we built it, read this blog post [pgai: Giving PostgreSQL Developers AI Engineering Superpowers](http://www.timescale.com/blog/pgai-giving-postgresql-developers-ai-engineering-superpowers).
 
-## Installation
+## Set up pgai
```
Comment (jgpruitt) [inline README.md:36]:
```suggestion
## Installation
```
Why not "Installation" like we have in pgvectorscale's readme? I think people default to looking for "Installation".

File: README.md [hunk 3]
```diff
@@ -17,35 +17,40 @@ pgai brings embedding and generation AI models closer to the database. With pgai
...
 
 The fastest ways to run PostgreSQL with the pgai extension are to:
 
-* [Use a pre-built Docker container](#use-a-pre-built-docker-container)
-* [Install from source](#install-from-source)
-* [Use a Timescale Cloud service](#use-a-timescale-cloud-service)
+1. Create your Timescale environment. Either:
+   * [Use a pre-built Docker container](#use-a-pre-built-docker-container).
+   * [Install from source](#install-from-source).
+   * [Use a Timescale Cloud service](#use-a-timescale-cloud-service).
```
Comment (jgpruitt) [inline README.md:43]:
```suggestion
1. Create your database environment. Either:
   * [Use a pre-built Docker container](#use-a-pre-built-docker-container).
   * [Install from source](#install-from-source).
   * [Use a Timescale Cloud service](#use-a-timescale-cloud-service).
```
Avoid "Timescale". For instance, "install from source" does not involve Timescale.

File: docs/advanced.md
Comment (jgpruitt) [inline docs/advanced.md:1]: Should we just remove this file?

File: docs/openai.md [hunk 4]
```diff
@@ -0,0 +1,592 @@
+# Use pgai with OpenAI
+
+This page shows you how to:
+
+- [Configure pgai for OpenAI](#configure-pgai-for-openai)
+- [Add AI functionality to your database](#usage)
+- [Follow advanced AI examples using data](./docs/advanced.md)
```
Comment (jgpruitt) [inline docs/openai.md:7]:
```suggestion
- [Follow advanced AI examples](#advanced-examples)
```

File: docs/ollama.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+This page shows you how to:
+
+- [Configure pgai for Ollama](#configure-pgai-for-ollama)
+- [Add AI functionality to your database](#usage)
+
+## Configure pgai for Ollama
+
+To use pgai with Ollama, Ollama must be running and network-accessible to your Timescale Cloud service or 
+self-hosted database.
```
Comment (jgpruitt) [inline docs/ollama.md:11]:
```suggestion
To use pgai with Ollama, Ollama must be running and network-accessible to your database.
```

File: docs/ollama.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+
+To specify the Ollama network address, Ollama functions in pgai use the `_host` parameter. 
+If you have not set `_host`, the `ai.ollama_host` config setting is missing. pgai defaults to 
+`http://localhost:11434`, and a warning is appended to to the log file.
```
Comment (jgpruitt) [inline docs/ollama.md:15]:
```suggestion
If you do not pass a `_host` argument, and the `ai.ollama_host` config setting is missing. pgai defaults to 
`http://localhost:11434`, and a warning is appended to the log file.
```

File: docs/ollama.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+
+- [Configure pgai for Ollama](#configure-pgai-for-ollama)
+- [Add AI functionality to your database](#usage)
+
+## Configure pgai for Ollama
+
+To use pgai with Ollama, Ollama must be running and network-accessible to your Timescale Cloud service or 
+self-hosted database.
+
+To specify the Ollama network address, Ollama functions in pgai use the `_host` parameter.
```
Comment (jgpruitt) [inline docs/ollama.md:13]:
```suggestion
To specify the Ollama network address, Ollama functions in pgai use the `_host` parameter. Alternatively, the `ai.ollama_host` config setting can used.
```

**Review by jgpruitt — CHANGES_REQUESTED** [review]

**Review by jgpruitt — APPROVED** [review]

**Pull request merged**